scanpipe/templates/scanpipe/error_list.html

```diff
@@ -52,6 +52,9 @@
               <td style="max-width: 550px;">
                 <pre class="log wrap p-0" style="max-height: 150px;"><code>{{ error.traceback }}</code></pre>
               </td>
+              <td style="max-width: 550px;">
+                <a class="has-text-danger modal-button" data-target="modal-report" aria-haspopup="true">Github</a>
```

Comment:
What about this instead?
```suggestion
                <a class="has-text-danger modal-button" data-target="modal-report" aria-haspopup="true">Report issue on GitHub</a>
```

Also can you attach screenshots that show how this looks?

Reply:
![image](https://user-images.githubusercontent.com/67731696/226833854-234c6152-cfbe-4b69-8fb6-7609b5ee8a15.png)
it is looking like this

Reply:
![image](https://user-images.githubusercontent.com/67731696/226834029-6169a8db-7255-443e-a701-3b3f9ed3bd9e.png)
it looks like this

Reply:
but if i add Report issue on GitHub it will take too much space

Reply:
![image](https://user-images.githubusercontent.com/67731696/227709203-e1f94b1f-1bd5-4ae2-be6a-d399141e9b39.png)